content-repo/test_data/bad-mdx-readme.md

```diff
@@ -568,9 +569,9 @@ This integration was integrated and tested with version xx of Proofpoint TAP v2
 </p>
 <h5>Context Example</h5>
 <pre>
-{
+\{
```

Comment:
[here](https://mdxjs.com/docs/troubleshooting-mdx/#could-not-parse-expression-with-acorn-error) nad everywhere